src/audio/drc/drc_math_lib/asin/c_fixpt/drc_asin_fixpt.c

```diff
@@ -12,45 +12,40 @@
 #include "stdint.h"
 
 /* Function Declarations */
-static int32_T asr_s32(int32_T u, uint32_T n);
-static int8_T scalar_cordic_asin(int8_T sinValue, int16_T nIters, const int8_T
+static int32_t asr_s32(int32_t u, uint32_t n);
+static int8_t scalar_cordic_asin(int8_t sinValue, int16_t nIters, const int8_t
   LUT[6]);
 
 /* Function Definitions */
 
  /*
  *-----------------------------------------------------------------------------
  *
- * Prototype       :static scalar_cordic_asin
- * Description     :Compute the inverse Sine using a CORDIC implementation.
- * ASR 32          : Arithmetic shift right  for 32 bit
- * Arithmetic bit right shift is performed for CORDIC iteration
+ * Prototype       :static int32_t asr_s32
+ * Description     :perform Arithmetic bit right shift on cordic iteration.
  * For angles less than pi/4, we initialize (x,y) along the x-axis [ see code variable - xShift,xDShift].
  * For angles greater than or equal to pi/4, we initialize (x,y) along the y-axis[see code variable - yShift,yDShift].
  * This improves the accuracy of the algorithm near the edge of the domain of convergence.
  * Variables j,k are derived from the Cordiac Iteration ( 1 to wordLength-1)
  * j = min(bitsll(i,1),maxShift) // bitsll logical left shift
  * k = min(i - 1,maxShift);
```

Comment:
Line 26~31 mainly describes the code implemented in scalar_cordic_asin(), right?
Maybe it is a better idea to have these comments along the code implementation, then it will be easier to follow the code and know what it does.